src/pages/home/home.html

```diff
@@ -42,14 +42,17 @@ <h1 text-center>{{ disciplina.titulo }}</h1>
         <h2 class="titulo">Média final</h2>
       </div>
 
-      <previsao-nota titulo="Necessário tirar na AP1" [nota]="disciplina.notas.notaParaAP1"></previsao-nota>
-      <previsao-nota titulo="Necessário tirar na AP2" [nota]="disciplina.notas.notaParaAP2"></previsao-nota>
-      <previsao-nota titulo="Necessário tirar na AP3" [nota]="disciplina.notas.notaParaAP3"></previsao-nota>
+      <previsao-nota *ngIf="disciplina.notas.notaParaAP1 > 0" titulo="Necessário tirar na AP1" [nota]="disciplina.notas.notaParaAP1"></previsao-nota>
```

Comment:
Pode ser também...

Mas o que eu estava sugerindo, era adicionar essa condição direto no component:

https://github.com/RodolfoSilva/DeVryCalc/blob/master/src/components/previsao-nota/previsao-nota.ts#L8

```
nota !== null && nota <= 10
```

Porque ele hoje já verifica se é diferente de nullo e menor ou igual a 10, no caso vc iria adicionar a verificação se nota for maior que 0.